### ISSUE TITLE
Enhancement: cronjob to renew letsencrypt certificate

### ISSUE THREAD

**Issue by ggeorgg** [ReinerNippes/nextcloud#24]
What do you think about adding a task which creates a cronjob to renew the letsencrypt certificate?

Cheers

**Comment by ggeorgg** [comment]
#25

**Comment by ReinerNippes** [comment]
I can test and merge this. 
My understanding was that certbot is refreshing the certificates automatically. It only stopped if one didn't update to the latest. version and letsencryted canceled an insecure challenge method.

**Comment by ggeorgg** [comment]
I just ran the playbook with this modification and it didn't work.
I did not run into any trouble before because I have not tested it before. It just got my attention when reading the code. 
But I think you are right, that certbot is refreshing the certificates automatically:
[https://certbot.eff.org/docs/using.html#renewing-certificates](https://certbot.eff.org/docs/using.html#renewing-certificates):

> Many of the certbot clients obtained through a distribution come with automatic renewal out of the box, such as Debian and Ubuntu versions installed through apt, CentOS/RHEL 7 through EPEL, etc. See Automated Renewals for more details.

So I think this can be closed.
